ecosystem/html_templates/webpage.html.jinja

```diff
@@ -0,0 +1,87 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Qiskit Ecosystem</title>
```

Comment:
You could add the description now: https://github.com/qiskit-community/ecosystem/issues/569. Frank gave a suggestion of what text to use.

Reply:
Done! :+1: